bats/tests/preferences/surface-invalid-args.bats

```diff
@@ -0,0 +1,61 @@
+load '../helpers/load'
+
+local_setup() {
+    if is_windows; then
+        skip "test not applicable on Windows"
+    fi
+}
+
+@test 'initial factory reset' {
+    factory_reset
+}
+
+can_set_vm_to_vz() {
+    if ! is_macos; then
+        false
+    else
+        version=$(/usr/bin/sw_vers -productVersion)
+        majorMinorVersion="${version%.*}"
```

Comment:
Please don't use camelCase for shell variables!

But the whole logic doesn't work because the return value does not include a patch version, if it is `0`. Sample values:

```
14.0
12.7
10.15.7
```

We should have a test for this logic, once it is moved to the `helpers`. The test can be done in a follow-up PR after the release.

Reply:
Written up at issue #5741